pkg/services/provisioning/dashboards/file_reader_test.go

```diff
@@ -248,6 +250,42 @@ func TestDashboardFileReader(t *testing.T) {
 				So(folderCount, ShouldEqual, 2)
 				So(dashCount, ShouldEqual, 2)
 			})
+
+			Convey("Sanity checker should collect info about duplicate uids and folder/title's", func() {
+				folderID, err := getOrCreateFolderID(cfg, fakeService, "sanity-check-folder")
+				So(err, ShouldBeNil)
+
+				identity := dashboardIdentity{folderID: folderID, title: "Grafana"}
+
+				cfg1 := &config{Name: "first", Type: "file", OrgID: 1, Folder: "sanity-check-folder", Options: map[string]interface{}{"path": containingUID}}
```

Comment:
This is more readable.
```suggestion
				cfg1 := &config{
					Name: "first", Type: "file", OrgID: 1, Folder: "sanity-check-folder", 
					Options: map[string]interface{}{"path": containingUID},
				}
```